### ISSUE TITLE
Can't find ~/.mycroft/skills/MycroftRoutineSkill/routines.json

### ISSUE THREAD

**Issue by sanderant** [ChristopherRogers1991/mycroft_routine_skill#6]
I have the skill running, and I have created a routine, but I don't see this file. Do I need to create the directory or is could it be stored somewhere else?

pi@mark_1:~ $ tree ./.mycroft
./.mycroft
└── identity

This is running 19.2.1b

Thanks

**Comment by ChristopherRogers1991** [comment]
It looks like you're logged in as the `pi` user. I'm assuming this is on a Mark 1 or picroft then, so the actual Mycroft process is probably running under the `mycroft` user. Can you run `sudo su - mycroft` to switch to that user, and then look for it in that user's home directory?

If this is not a Mark 1 or picroft, can you tell me how you installed it?

**Comment by sanderant** [comment]
You are correct this is a Mark 1, and it was installed when I got it, so it was likely from a custom image. Su to become the mycroft user does indeed show the directory. I would add this note to the documentation, unless I missed it, for n00bs like myself. 

Thanks so much for this great skill I hope it gets added to the marketplace.
